src/current/cockroachcloud/physical-cluster-replication.md

```diff
@@ -0,0 +1,286 @@
+    ~~~
+
+    If you're creating clusters in AWS or Azure, you must start the primary and standby clusters in different regions.
+
+{{site.data.alerts.callout_success}}
+We recommend [enabling Prometheus metrics export]({% link cockroachcloud/export-metrics.md %}) on your cluster before starting a PCR stream. For details on metrics to track, refer to [Monitor the PCR stream](#step-3-monitor-the-pcr-stream).
+{{site.data.alerts.end}}
+
+### Step 2. Start the PCR stream
+
+{{site.data.alerts.callout_info}}
+We recommend using an empty standby cluster when starting PCR. When you initiate the PCR stream, CockroachDB {{ site.data.products.cloud }} will take a full cluster backup of the standby cluster, delete all data from the standby, and then start the PCR stream, which ensures that the standby will be fully consistent with the primary during PCR.
+{{site.data.alerts.end}}
+
+With the primary and standby clusters set up, you can now start a PCR stream.
+
+1. Send a `POST` request to the `/v1/physical-replication-streams` endpoint to start the PCR stream:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request POST --url 'https://cockroachlabs.cloud/api/v1/physical-replication-streams' --header "Authorization: Bearer api_secret_key" --json '{"primary_cluster_id": "primary_cluster_id","standby_cluster_id": "standby_cluster_id"}'
+~~~
+
+Replace:
+
+- `api_secret_key` with your API secret key.
+- `primary_cluster_id` with the cluster ID returned after creating the primary cluster.
+- `standby_cluster_id` with the cluster ID returned after creating the standby cluster.
+
+You can find the cluster IDs in the cluster creation output, or in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`. The ID will resemble `ad1e8630-729a-40f3-87e4-9f72eb3347a0`.
+
+{{site.data.alerts.callout_info}}
+Once you have started PCR, the standby cluster cannot accept writes and reads, therefore the [Cloud Console]({% link cockroachcloud/cluster-overview-page.md %}) and SQL shell will be unavailable prior to failover.
+{{site.data.alerts.end}}
+
+You will receive a response similar to:
+
+~~~ json
+{
+  "id": "c3d35c84-a4ea-41b3-8452-553c5ded3b85",
+  "status": "STARTING",
+  "primary_cluster_id": "3fabc29e-5ced-48d9-b31e-000000000000",
+  "standby_cluster_id": "f9b1d580-9be3-47f8-ac28-000000000000",
+  "created_at": "2025-05-01T18:57:50.038137Z"
+}
+~~~
+
+- `"id"`: The PCR stream's job ID.
+- `"status"`: The status of the PCR stream. For descriptions, refer to [Status](#status).
+- `"primary_cluster_id"`, `"standby_cluster_id"`: The cluster IDs of the primary and standby clusters.
+- `"created_at"`: The time at which the PCR stream was created.
```

Comment:
annoying q but is this is a `TIMESTAMP{,TZ}` in the SQL sense or just, like, "a string in some JSON that means a time" ? if former would it make sense to link to `TIMESTAMP` docs?